### PR TITLE
Fix a typo in comment

### PR DESCRIPTION
**Pull request by bernd-edlinger**

### CODE REVIEW

File: crypto/ec/ecdsa_ossl.c [hunk 1]
```diff
@@ -298,7 +298,7 @@ ECDSA_SIG *ossl_ecdsa_sign_sig(const unsigned char *dgst, int dgst_len,
         }
         if (BN_is_zero(s)) {
             /*
-             * if kinv and r have been supplied by the caller don't to
+             * if kinv and r have been supplied by the caller don't do
              * generate new kinv and r values
```
Comment (kroeckx) [inline crypto/ec/ecdsa_ossl.c]: remove the do?

Comment (bernd-edlinger) [reply crypto/ec/ecdsa_ossl.c]: maybe, I am not a native English speaker...
should the sentence start with capital letter, and end with a period?

**Comment by t-j-h** [comment]
Code comments in general do not have to be "correct" english in terms of punctuation and capitalisation

File: crypto/ec/ecdsa_ossl.c (hunk 1 above)
Comment (bernd-edlinger) [reply crypto/ec/ecdsa_ossl.c]: ok, so I just remove the "do".

File: crypto/ec/ecdsa_ossl.c [hunk 2]
```diff
@@ -298,7 +298,7 @@ ECDSA_SIG *ossl_ecdsa_sign_sig(const unsigned char *dgst, int dgst_len,
         }
         if (BN_is_zero(s)) {
             /*
-             * if kinv and r have been supplied by the caller don't to
+             * if kinv and r have been supplied by the caller don't
```
Comment (levitte) [inline crypto/ec/ecdsa_ossl.c]: Not to be contrary, but wouldn't the flow of the message be better with a comma after "called"?

(I kinda sorta agree with @t-j-h, but only in so far that the sentence doesn't become confusing, or its flow diminished)

Comment (bernd-edlinger) [reply crypto/ec/ecdsa_ossl.c]: ah okay, you mean a comma after "caller"?

Comment (levitte) [reply crypto/ec/ecdsa_ossl.c]: Argh, yes!  (early morning here, that'l be my excuse)

**Comment by bernd-edlinger** [comment]
Merged. Thanks!

**Pull request closed without merging**